src/orders/model/OrdersQueryRequest.ts

```diff
@@ -2,5 +2,5 @@ import PaginationQueryRequest from '../../model/PaginationQueryRequest';
 import OrderStatus from './OrderStatus';
 
 export default interface OrdersQueryRequest extends PaginationQueryRequest {
-  status?: OrderStatus;
+  status?: OrderStatus | OrderStatus[];
```

Comment:
@ryankazokas I agree, but didn't do that because of the breaking change.  

Should I open a github issue for it so we don't forget about it?

Reply:
Yeah, that's fine @mb-ca . Usually we look for stuff like this to improve on when we are incrementing a major version, but it might not be bad to track.